packages/dtslint/test/dt-header.test.ts

```diff
@@ -0,0 +1,168 @@
+import { ESLintUtils } from "@typescript-eslint/utils";
+
+import * as dtHeader from "../src/rules/dt-header";
+
+const ruleTester = new ESLintUtils.RuleTester({
+  parser: "@typescript-eslint/parser",
+});
+
+ruleTester.run("no-const-enum", dtHeader, {
+  invalid: [
+    {
+      code: ``,
+      errors: [
+        {
+          column: 2,
+          data: {
+            expected: "/\\/\\/ Type definitions for (non-npm package )?/",
+          },
+          line: 1,
+          messageId: "parseError",
+        },
+      ],
+      filename: "types/abc/index.d.ts",
```

Comment:
can you give a meaningful name instead of `abc`, since eslint doesn't let you name the test cases explicitly?